react/client/src/components/FlowNavigation.tsx

```diff
@@ -17,29 +19,43 @@ const useStyles = makeStyles(() =>
 );
 
 interface IFlowNavigation {
-  goNextPage: () => void;
-  goBackPage: () => void;
+  onNext?: () => void;
+  onBack?: () => void;
```

Comment:
Navigation can now take optional functional overrides for the prev/next button -- otherwise it'll just directly handle step routing